src/lib/factory.js

```diff
@@ -73,6 +74,16 @@ class PluginFactory extends EventEmitter2 {
     this.ready = true
   }
 
+  _accountToUsername (accountUrl) {
```

Comment:
This method already exists: https://github.com/interledgerjs/ilp-plugin-bells/blob/cfee4821a7f8c2814365d74622846d0895ad452e/src/lib/ledger-context.js#L60-L66

Would be available as `this.ledgerContext.accountUriToName` in this context.